### ISSUE TITLE
Scale of bar graph does not change 

### ISSUE THREAD

**Issue by data-compute-geek** [bmabey/pyLDAvis#226]
I am trying to use pyLDAvis to visualize the result of a topic modeling project. I notice that the scale of the axis for the frequency bar does not change when the length of the bars change as I slide the lambda dial. The visualization can be accessed here  https://seanlog.com/tmp/lda_visualization_tutorial.html#topic=4&lambda=0&term=

For example, for topic 4, the frequency of christian is >2,000 when lambda is set at 0, and changes to ~ 1,500 when lambda is changed to 0.2.
![image](https://user-images.githubusercontent.com/11891624/167319602-c790c74f-b03c-45a4-91d9-74b3517686fc.png)
![image](https://user-images.githubusercontent.com/11891624/167319612-18e589b7-e7a3-4747-9d35-08f57409f0bc.png)
My understanding is that changing the lambda only changes the order of the word, but not the overall or topic-specific frequencies. Shouldn't the scale of the axis for the bars change as lambda changes so as to keep the frequencies constant? 

I used pyLDAvis.gensim_models.prepare to generate the visualization, and the code used to generate the visualization is available at https://github.com/data-compute-geek/pyLDAvis/blob/main/topic-modeling_tutorial.py

**Comment by chschroeder** [comment]
Same here. I am using pyLDAvis within a notebook:

```
import pyLDAvis
import pyLDAvis.sklearn

pyLDAvis.enable_notebook()

pyLDAvis.sklearn.prepare(topic_model, x_train, vectorizer)
```

where topic_model is an instance of `sklearn.decomposition.LatentDirichletAllocation` and vectorizer is a `CountVectorizer`.

**Comment by ldong87** [comment]
This happens to me too. The frequency ticks are now static somehow.
